src/services/utilities.ts

```diff
@@ -469,6 +469,38 @@ namespace ts {
         }
     }
 
+    /**
+     * Get the corresponding JSDocTag node if the position is in a jsDoc commet
+     */
+    export function getJsDocTagAtPosition(sourceFile: SourceFile, position: number): JSDocTag {
+        let node = ts.getTokenAtPosition(sourceFile, position);
+        let jsDocComment: JSDocComment;
+        if (node.jsDocComment) {
+            jsDocComment = node.jsDocComment;
+        }
+        else {
+            while (node) {
+                if (node.kind === SyntaxKind.VariableStatement ||
+                    node.kind === SyntaxKind.FunctionDeclaration ||
+                    node.kind === SyntaxKind.Parameter) {

```

Comment:
My understanding is that you're trying to find the declaration that a token belongs to. A better test might be `!isToken(node)`